### ISSUE TITLE
Drag and Drop Groups empty

### ISSUE THREAD

**Issue by jaykijay** [zammad/zammad#1591]
### Infos:

* Used Zammad version: Zammad.com Demo and my own with Debian 9, latest Development (2.1)
* Used Zammad installation source: (source, package, ...) -
* Operating system: -
* Browser + version: -

### Expected behavior:
Dragging Tickets down should show Groups with Useres in it to assign Tickets to them.

### Actual behavior:
Groups are displayed as empty. No Users are shown. But there ARE Users/Agents in the Helpdesk Group!

### Steps to reproduce the behavior:
Create a Group "Helpdesk"
Create a Role "Helpdesk" and asign the full rights to the Group Helpdesk (under agent) to it.
Asign the Role /Rights "Helpdesk" to Users, if via LDAP or manual doesn't matter.
Choose any open ticket, drop it to the Bottom, it will show the following Screenshot:
![grafik](https://user-images.githubusercontent.com/24495677/32100865-cd2b5084-bb15-11e7-8c38-264a35401cbf.png)

**Comment by jaykijay** [comment]
Update: It's still broken with latest Development (2.2) even on your Demo System at Zammad.com.

**Comment by jaykijay** [comment]
Still broken everywhere.

**Comment by martini** [comment]
Hi @jaykijay just tried to verify your issue. It's not clear to me what the issue is. 

If there is a group without agents to select (0 Agents), you should be able to move tickets into (just without assigning an agent). This is also working on my side.

You think different?

**Comment by jaykijay** [comment]
So the Problem is, that i created Agents in Group Helpdesk, but it's still shown as if there were no Useres in this Group.
You can simply test this on https://bugme.zammad.com. or create this scenario as i described it in the first Post.
If you need Access Rights to this Instance please tell me.

**Comment by thorsteneckel** [comment]
Hi @jaykijay - we are now ready to work on the issue. However, the Zammad instance got removed already 😱 Could you please provide a detailed step by step description how to reproduce this in a newly created Zammad instance?

**Comment by MrGeneration** [comment]
Hey,

I could reproduce this in my production system @2.4 - but it's a bit different here: It shows me mostly "1 agent" inside:

![image](https://user-images.githubusercontent.com/6549061/39749343-bdef9c76-52b2-11e8-9776-607dc80f029e.png)

Let's take "Bühne" as example - when I want to trag it into the group, I can only see me as agent. If I take a look into "Spam"  then I can see the correct 2 agents having access to it. I've taken a look into it and think I found the issue here: For Spam I've set the group rights manually in addition to assigning the group. I did that earlier in 2.2 as I found that for some situations (especially with Admins) the group assignment does not work correctly. It's the only difference I can spot and think as cause of.

![image](https://user-images.githubusercontent.com/6549061/39749582-5f2671d2-52b3-11e8-8932-abbae501f5d8.png)


![image](https://user-images.githubusercontent.com/6549061/39749645-8f0f54d6-52b3-11e8-9ab2-6646d15570b9.png)


I hope that this helps :)
